interface/modules/custom_modules/oe-module-weno/templates/weno_fragment.php

```diff
@@ -0,0 +1,58 @@
+<?php 
+require_once(dirname(__DIR__, 4) . "/globals.php");
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Twig\TwigContainer;
+
+$twig = new TwigContainer(null, $GLOBALS['kernel']);
```

Comment:
is above $twig being used anywhere. If not, then remove it and the use TwigContainer line.